popups/cloud-games/light.css

```diff
@@ -9,11 +9,10 @@ body {
   color: #222;
 }
 
-.title {
-  color: #222;
-}
-
-.title a {
+.loading,
+.title,
+.title a,
+.username-list > a {
```

Comment:
Usernames should be blue.
Screenshot from v1.10.0:
![image](https://user-images.githubusercontent.com/17484114/124395577-40eaa480-dcdb-11eb-9f2b-05e974a902c5.png)